.github/workflows/split-docker-build.yml

```diff
@@ -0,0 +1,254 @@
+name: Build Docker images
+
+on:
+    push:
+        tags:
+            - '*.*.*'
+        branches:
+            - main
+    pull_request:
+        paths:
+            - 'Dockerfile.*'
+            - 'dockerV2/**'
+            - '.github/workflows/split-docker-build.yml'
+    workflow_dispatch:
+
+jobs:
+    build:
+        name: Build Docker Images
+        runs-on: ubuntu-latest
+        steps:
+            -   name: Check out repository
+                uses: actions/checkout@v2
+
+            -   name: Get short commit SHA
+                id: sha
+                run: echo "::set-output name=short::${GITHUB_SHA::7}"
+
+            -   name: Set build platforms
+                id: platform
+                run: |
+                    if [ "${{ github.ref_type }}" = tag ]; then
+                        echo "::set-output name=targets::linux/amd64,linux/arm64"
+                    else
+                        echo "::set-output name=targets::linux/amd64"
+                    fi
+
+            -   name: Check short commit SHA and build targets
+                run: |
+                    echo ${{ steps.sha.outputs.short }}
+                    echo ${{ steps.platform.outputs.targets }}
+
+            -   name: Docker resotobase meta
+                id: basemeta
+                uses: docker/metadata-action@v3
+                with:
+                    images: |
+                        someengineering/resotobase
+                        ghcr.io/someengineering/resotobase
+                    flavor: |
+                      latest=auto
+                    tags: |
+                        type=pep440,pattern={{version}}
+                        type=pep440,pattern={{major}}.{{minor}}
+                        type=pep440,pattern={{major}}
+                        type=sha,prefix=
+                        type=edge
+                    labels: |
+                        org.opencontainers.image.title=resotobase
+                        org.opencontainers.image.description=Resoto base image
+                        org.opencontainers.image.vendor=Some Engineering Inc.
```

Comment:
Nice!  We should also add `org.opencontainers.image.created`, `org.opencontainers.image.revision`, `org.opencontainers.image.source`, `org.opencontainers.image.url`, and `org.opencontainers.image.version`.

Reply:
This is just the list of labels we override. The meta action adds the rest. We're currently setting:
```
      "org.opencontainers.image.title": "resotobase",
      "org.opencontainers.image.description": "Resoto base image",
      "org.opencontainers.image.url": "https://github.com/someengineering/resoto",
      "org.opencontainers.image.source": "https://github.com/someengineering/resoto",
      "org.opencontainers.image.version": "8d402ef",
      "org.opencontainers.image.created": "2022-01-20T18:05:52.062Z",
      "org.opencontainers.image.revision": "8d402ef44405a507eda6b5f1663d74d55d0afb08",
      "org.opencontainers.image.licenses": "Apache-2.0",
      "org.opencontainers.image.vendor": "Some Engineering Inc."
```